### ISSUE TITLE
Improve color contrast for slack icon in dark mode

### ISSUE THREAD

**Issue by yuanLeeMidori** [Seneca-CDOT/telescope#2050]
**What happened**:
![image](https://user-images.githubusercontent.com/50813726/112927537-2c4aaf00-90e3-11eb-8a84-bef3a0e9a122.png)
In about footer, the slack icon is hard to see in dark mode.
**What should have happened**:

**How to reproduce it (as precise as possible)**:

**Anything else we need to know?**:

**Environment**:

- OS:
- Browser:
